app/tests/test_views.py

```diff
@@ -0,0 +1,53 @@
+""" Tests for views.py """
+from app import app
+import unittest
+import json
+
+
+class TestRouteCases(unittest.TestCase):
+    """ Test Cases """
+    def setUp(self):
+        """ Instantiate test client """
+        self.app = app.test_client()  # test client
+        # Sample user data
+        self.sample = {
+            'username': 'artorious',
+            'email': 'arthurngondo@email.com',
+            'password': 123454321,
+            'confirm_password': 123454321,
+            'first_name': 'Arthur',
+            'last_name': 'Ngondo'
+            }
+
+    def test_register_status(self):
+        """ Test endpoint sends HTTP POST request to the application."""
+        result = self.app.post(
+            '/api/v1/register',
+            data=json.dumps(self.sample),
+            headers={'content-type': 'application/json'}
+        )
+        self.assertEqual(result.status_code, 201)
+
+    def test_register_Success_operation_message(self):
```

Comment:
this could be one test with `test_register_status` you just have to add another assertEqual for the data